riak/transports/pbc/connection.py

```diff
@@ -165,7 +165,7 @@ def _ssl_handshake(self):
 
                     return True
                 except ssl.SSLError as e:
-                    raise SecurityError(e.library + ": " + e.reason)
```

Comment:
If you could try it on 2.7.9, say, that would be helpful.  Then we'd know it works everywhere.  Have you grepped for `.library`? We should change it everywhere to be consistent.